common/config.py

```diff
@@ -152,6 +152,9 @@ def __init__(self):
         # OSes
         self.allowed_oses = set(os.getenv("ALLOWED_OSES", "RHEL").split(","))
 
+        # Set to true if connecting to VMaaS aware of multiple Satellite organizations
+        self.multi_org_vmaas = strtobool(os.getenv("MULTI_ORG_VMAAS", "FALSE"))
```

Comment:
Is there an option to have this auto-detected?

Reply:
Not currently, but good idea actually to implement it

Reply:
Let's wait with merging this one, I'll try to implement support for it first